platform/stm32f4xx/rules.mk

```diff
@@ -14,16 +14,16 @@ ARM_CPU := cortex-m4
 
 # TODO: integrate better with platform/stm32f4xx/CMSIS/stm32f4xx.h
 ifeq ($(STM32_CHIP),stm32f407)
-GLOBAL_DEFINES += STM32F40_41xxx 
+GLOBAL_CFLAGS=-DSTM32F40_41xxx=1
```

Comment:
It's preferable to use GLOBAL_DEFINES += STM32F40_41xxx=1 here instead of CFLAGS. GLOBAL_CFLAGS doesn't work exactly how you think it does, and the defines tracks things better by putting in a header file instead of on the command line.

Reply:
Interesting GLOBAL_DEFINES didn't work, do I need to include that header somehow?